internal/sdkv2provider/schema_cloudflare_access_application.go

```diff
@@ -896,7 +896,7 @@ func convertSaasSchemaToStruct(d *schema.ResourceData) *cloudflare.SaasApplicati
 			customClaims, _ := d.Get("saas_app.0.custom_claim").([]interface{})
 			for _, customClaims := range customClaims {
 				claimsAsMap := customClaims.(map[string]interface{})
-				SaasConfig.CustomClaims = append(SaasConfig.CustomClaims, convertOIDCClaimSchemaToStruct(claimsAsMap))
+				*SaasConfig.CustomClaims = append(*SaasConfig.CustomClaims, convertOIDCClaimSchemaToStruct(claimsAsMap))
```

Comment:
These changes were not necessarily in scope but I had to make them after bumping the version of `cloudflare-go`.

Reply:
thanks but could you back these out? we have them sorted in a different branch and want to avoid merge conflicts if we can.

Reply:
Will do!